pkg/actions/install.go

```diff
@@ -36,20 +36,27 @@ func InstallCommand(c *cli.Context) {
 		"docker.io/eclipse/codewind-pfe-amd64:" + tag,
 		"docker.io/eclipse/codewind-performance-amd64:" + tag,
 	}
+
+	dockerClient, dockerErr := utils.NewDockerClient()
```

Comment:
Do you think there could be benefit to adding a comment here to explain what the docker client is doing?